README.md

```diff
@@ -47,7 +49,10 @@ public async Task GetMyData()
            .AddHandler(new MyHandler2())
            .AddHandler(new MyHandler1());
            
-    var client = new HttpClient(builder.Build());
+    var tracer = builder.Build();
+    tracer.Verbosity = LogLevel.Information;
```

Comment:
Should we create a constructor that takes the verbosity level? It will be optional since we have